### PR TITLE
Load More Button

### LINKED ISSUE #40: Load More Button
## Description

image

## Learning Objectives

- Objective 1

## Todo

These points are a rough guideline. Please feel free to discuss with others on the team about the best way to design this component!

- [ ] Todo 1

## Useful Links

- Link 1

### PR DESCRIPTION
**Pull request by lizageorge**
## Summary

Creates Load More Button for Investments page
Closes #40 

## Changes

- Load More Button
- Disabled and Active Styling

## Screenshots

image
## Requests / Responses

https://user-images.githubusercontent.com/20449375/224182886-9cf80246-2e1b-4675-b6e8-c44b931071c5.mov

### CODE REVIEW

File: src/components/Buttons/LoadMoreButton.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import type { FC } from 'react'
+import clsx from 'clsx'
+
+interface LoadMoreButtonProps {
+  onClick: () => void
+  disabled: boolean
+  children?: React.ReactNode
```
Comment (zorazrr) [inline src/components/Buttons/LoadMoreButton.tsx:7]: Do we need this?

File: src/components/Buttons/LoadMoreButton.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+const LoadMoreButton: FC<LoadMoreButtonProps> = ({ onClick, disabled }) => {
+  return (
+    <button
+      className={clsx(
+        'font-klima text-[28px] font-medium',
+        'rounded border-2 border-solid border-clementine bg-white',
+        'm-6 px-7 py-3.5',
+        'shadow-[-8px_8px_0px_0px] shadow-clementine',
+        'active:shadow-[-12px_12px_0px_0px] active:shadow-clementine',
```
Comment (zorazrr) [inline src/components/Buttons/LoadMoreButton.tsx:18]: When we hover, can we have the same style as well?

**Review by zorazrr — CHANGES_REQUESTED** [review]
Looks good! Just some small changes.

File: src/components/Buttons/LoadMoreButton.tsx (hunk 1 above)
Comment (lizageorge) [reply src/components/Buttons/LoadMoreButton.tsx:7]: I'm a bit confused by what you mean - are you asking if we need this interface or if we need props at all? Or a specific field within it?
This structure is a simplified version of this tutorial - [https://www.twilio.com/blog/intro-custom-button-component-typescript-react](https://www.twilio.com/blog/intro-custom-button-component-typescript-react)

Comment (lizageorge) [reply src/components/Buttons/LoadMoreButton.tsx:7]: Nwm, turns out it is is unnecessary (the children field)

**Review by zorazrr — APPROVED** [review]
Looks good!

**Pull request merged**